training/Dockerfile.gpu

```diff
@@ -11,20 +11,27 @@ RUN apt-get update && \
 
 RUN pip3 install -U pip
 
-RUN pip3 --no-cache-dir install \
-        tensorflow-gpu \
-        tensorflowjs
-
 WORKDIR /
 
 RUN git clone --depth 1 https://github.com/tensorflow/tensorflow.git --branch r1.7
 
 CMD python3 /tensorflow/tensorflow/examples/image_retraining/retrain.py \
         --image_dir /data/images \
-        --how_many_training_steps=10000 \
+        --how_many_training_steps=4000 \
```

Comment:
Just curious what the affect is of training less than before. Does the model still converge at this training step? Assuming you did a end to end test on these new docker files to make sure the output is correct?

Reply:
I run another training with this configuration on our training dataset and confirmed that the default number of training steps (4000) provides good convergence. I could remove this line since it's the default value, but I kept it because a good parameter value really depends on the number of training images and objects to classify.